### PR TITLE
Fixed typos and out of order items

### PR DESCRIPTION
**Pull request by george-hoffer**
https://www.mulesoft.org/jira/browse/DOCS-4593

### CODE REVIEW

File: modules/getting-started/pages/glossary.adoc [hunk 1]
```diff
@@ -37,13 +37,13 @@ Anypoint Platform™::
...
 
 Anypoint Runtime Fabric::
 A container service that automates the configuration, coordination, and management (orchestration), and deployment of your Mule applications and gateways. Runtime Fabric runs on customer-managed infrastructure on AWS, Azure, virtual machines (VMs) or bare-metal servers.
 
 Anypoint Runtime Manager (Runtime Manager)::
-A console where you deploy and manage applications built with any Mule. Deploy to servers in the cloud (currently handled by CloudHub) or on premises. s the console in Anypoint Platform or download it as a standalone program that runs in a local server. Previously referred to as ARM.
+A console where you deploy and manage applications built with any Mule. Deploy to servers in the cloud (currently handled by CloudHub) or on premises. Access the console the console in Anypoint Platform or download it as a standalone program that runs in a local server. Previously referred to as ARM.
```
Comment (valkyrie69) [inline modules/getting-started/pages/glossary.adoc]: Extra "the console"

File: modules/getting-started/pages/glossary.adoc [hunk 2]
```diff
@@ -62,7 +62,10 @@ API Manager::
 Part of the Anypoint Platform that allows you to design and manage your APIs. It includes the [API Designer], which allows you to write a [RAML] definition of your API. It also includes tools to monitor usage metrics, apply [policies] and to expose interactive documentation through an [API Portal]
```
Comment (valkyrie69) [inline modules/getting-started/pages/glossary.adoc]: I'm not sure why all those brackets are there. ? [API Designer] [RAML] etc. Maybe it's intentional.

File: modules/getting-started/pages/glossary.adoc [hunk 3]
```diff
@@ -77,23 +80,23 @@ The network allows consumers from other parts of the business to discover and us
...
+component::
+1. In Anypoint Platform, a POJO, Spring bean, Java bean, or a script such as Groovy, Ruby, Python, or JavaScript that contains the business logic for processing messages passing through Mule. These components typically take the whole message - or just the payload - as input. They return an object that becomes the message payload for the next element in the message processor chain.
+
+2. Generically, an architecturally specific portion of a software package.
+
 configuration builder:: 
 A class that parses a configuration file. The default configuration builder is the `org.mule.config.MuleXmlConfigurationBuilder` class, which parses a Mule XML configuration file.
 
+connector::
+A self-contained component that allows you to integrate Mule applications with the APIs of other, external applications, such as Salesforce, CMIS, and Twitter. A set of connectors is included in Mule version 4. Other connectors created by MuleSoft or third parties are available on Anypoint Exchange.
```
Comment (valkyrie69) [inline modules/getting-started/pages/glossary.adoc]: I would just say Mule 4 instead of Mule version 4, but feel free to disregard.

**Review by valkyrie69 — COMMENTED** [review]
Thanks for improving the quality George!

File: modules/getting-started/pages/glossary.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/getting-started/pages/glossary.adoc]: Doh. Fixed.

File: modules/getting-started/pages/glossary.adoc (hunk 2 above)
Comment (george-hoffer) [reply modules/getting-started/pages/glossary.adoc]: Commented out the sentences starting with "It includes" -- the sentences thereafter apply to the API Manager prior to Nov 2017 and are not relevant now.

File: modules/getting-started/pages/glossary.adoc (hunk 3 above)
Comment (george-hoffer) [reply modules/getting-started/pages/glossary.adoc]: Agree. Fixed.

File: modules/getting-started/pages/glossary.adoc (hunk 2 above)
Comment (valkyrie69) [reply modules/getting-started/pages/glossary.adoc]: Ah, okay.

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**